pkg/tfbridge/detailed_diff.go

```diff
@@ -482,6 +514,7 @@ func MakeDetailedDiffV2(
 	tfs shim.SchemaMap,
 	ps map[string]*SchemaInfo,
 	priorProps, props, newInputs resource.PropertyMap,
+	replaceOverride *bool,
```

Comment:
Perhaps this could use a quick doc comment what's the high-level intent of this (nil, ref(true), ref(false)), for a bit of redundancy.